README.md

```diff
@@ -18,18 +18,21 @@ This repository contains all the 0x developer tools written in TypeScript. Our h
+| [`@0xproject/assert`](/packages/assert)                                                | [![npm](https://img.shields.io/npm/v/@0xproject/assert.svg)](https://www.npmjs.com/package/@0xproject/assert)                                     | Type and schema assertions used by our packages                    |
+| [`@0xproject/connect`](/packages/connect)                                              | [![npm](https://img.shields.io/npm/v/@0xproject/connect.svg)](https://www.npmjs.com/package/@0xproject/connect)                                   | A Javascript library for interacting with the standard relayer api |
+| [`@0xproject/json-schemas`](/packages/json-schemas)                                    | [![npm](https://img.shields.io/npm/v/@0xproject/json-schemas.svg)](https://www.npmjs.com/package/@0xproject/json-schemas)                         | 0x-related json schemas                                            |
+| [`@0xproject/subproviders`](/packages/subproviders)                                    | [![npm](https://img.shields.io/npm/v/@0xproject/subproviders.svg)](https://www.npmjs.com/package/@0xproject/subproviders)                         | Useful web3 subproviders (e.g LedgerSubprovider)                   |
+| [`@0xproject/tslint-config`](/packages/tslint-config)                                  | [![npm](https://img.shields.io/npm/v/@0xproject/tslint-config.svg)](https://www.npmjs.com/package/@0xproject/tslint-config)                       | Custom 0x development TSLint rules                                 |
+| [`@0xproject/types`](/packages/types)                                                  | [![npm](https://img.shields.io/npm/v/@0xproject/types.svg)](https://www.npmjs.com/package/@0xproject/types)                                       | Shared type declarations                                           |
+| [`@0xproject/utils`](/packages/utils)                                                  | [![npm](https://img.shields.io/npm/v/@0xproject/utils.svg)](https://www.npmjs.com/package/@0xproject/utils)                                       | Shared utilities                                                   |
+| [`@0xproject/web3-wrapper`](/packages/web3-wrapper)                                    | [![npm](https://img.shields.io/npm/v/@0xproject/web3-wrapper.svg)](https://www.npmjs.com/package/@0xproject/web3-wrapper)                         | Web3 wrapper                                                       |
+| [`web3-typescript-typings`](/packages/web3-typescript-typings)                         | [![npm](https://img.shields.io/npm/v/web3-typescript-typings.svg)](https://www.npmjs.com/package/web3-typescript-typings)                         | Web3 typescript typings                                            |
```

Comment:
can we alphabetize?

Reply:
Initially my intention was to subjectively sort them by importance.
Most people are interested in 0x.js or contracts or connect. Almost none are gonna use our dev-utils. But maybe alphabetical order is a good idea